Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -710,6 +713,9 @@ def __getSingleMetadata( self, path ):
           metadataDict['Unavailable'] = 1
         else:
           metadataDict['Unavailable'] = 0
+        if attributeDict['user.status'] == 'ONLINE_AND_NEARLINE':
```

Comment:
https://github.com/DIRACGrid/DIRAC/commit/9f7f49bdd7d357f72fea966802b2410e8ef418df#diff-b5b53b92cedb0af1928db29712fc47df

In this PR XROOT has this metadata added 'for compatibility with SRM Storage' (migrated is not there though). So for that reason I would keep it in there, unless the XROOT has it for another reason and we don't need it anymore.